stackit/internal/services/cdn/distribution/resource.go

```diff
@@ -142,6 +143,7 @@ func (r *distributionResource) Metadata(_ context.Context, req resource.Metadata
 }
 
 func (r *distributionResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	backendOptions := []string{"http"}
 	resp.Schema = schema.Schema{
 		MarkdownDescription: features.AddBetaDescription("CDN distribution data source schema."),
 		Description:         "CDN distribution data source schema.",
```

Comment:
Commonly we add the available options to the attribute description as well, so it shows in the generated documentation in the tf registry.

```suggestion
		MarkdownDescription: features.AddBetaDescription("CDN distribution data source schema. " + utils.SupportedValuesDocumentation(backendOptions)),
		Description:         "CDN distribution data source schema. " + utils.SupportedValuesDocumentation(backendOptions),
```